RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate REST API - getAvatarSuggestion to Typescript

## Proposed changes

Migrate REST API - getAvatarSuggestion to Typescript

## Issue(s)	


## How to test or reproduce

- Logged in `open.rocket.chat`, you need to upload more than 1 photo to your profile
- Then, when you open the `ProfileView` should appear suggestions to your avatar

## Screenshots

## Types of changes




## Checklist



## Further comments
